swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -4773,7 +4798,6 @@ components:
         This card account report contains transactions resulting from the query parameters.
       type: object
       required:
-        - booked
```

Comment:
Relevant Change: booked is not mandatory any more. 
Adoption approach: Follow the change